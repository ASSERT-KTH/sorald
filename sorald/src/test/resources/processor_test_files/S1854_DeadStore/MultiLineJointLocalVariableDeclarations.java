/*
An assortment of multiline joint local variable declarations.
 */

public class MultiLineJointLocalVariableDeclarations {

    public void multiLineJointDeclarationWithDeadStore() {
        int veryLongVariableName = 1,
            otherLongVariableName = 2,
            finalLongVariableName = 3; // Noncompliant
        System.out.println(veryLongVariableName);
        System.out.println(otherLongVariableName);
    }

    public void multiLineJointDeclarationWithMissingInitializer() {
        int veryLongVariableName = 1,
                otherLongVariableName,
                finalLongVariableName = 3; // Noncompliant
        System.out.println(veryLongVariableName);
        otherLongVariableName = 2;
        System.out.println(otherLongVariableName);
    }
}