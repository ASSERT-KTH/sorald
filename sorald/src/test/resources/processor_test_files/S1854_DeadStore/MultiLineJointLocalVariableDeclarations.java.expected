/*
An assortment of multiline joint local variable declarations.
 */

public class MultiLineJointLocalVariableDeclarations {

    public void multiLineJointDeclarationWithDeadStore() {
        int veryLongVariableName = 1;
        int otherLongVariableName = 2;
        System.out.println(veryLongVariableName);
        System.out.println(otherLongVariableName);
    }

    public void multiLineJointDeclarationWithMissingInitializer() {
        int veryLongVariableName = 1;
        int otherLongVariableName;
        System.out.println(veryLongVariableName);
        otherLongVariableName = 2;
        System.out.println(otherLongVariableName);
    }
}